### PR TITLE
ref(monitors): Introduce layout-thirds into monitors

### PR DESCRIPTION
**Pull request by davidenwang**
Refactoring to use thirds layout, most pages will look unchanged.

Before:
image

After:
image

### CODE REVIEW

File: static/app/views/monitors/details.tsx [hunk 1]
```diff
@@ -44,23 +45,26 @@ class MonitorDetails extends AsyncView<Props, State> {
...
-        <MonitorIssues monitor={monitor} orgId={this.props.params.orgId} />
-
-        <Panel>
-          <PanelHeader>{t('Recent Check-ins')}</PanelHeader>
-
-          <MonitorCheckIns monitor={monitor} />
-        </Panel>
+        <Layout.Body>
+          <Layout.Main fullWidth>
+            <MonitorHeader
```
Comment (davidenwang) [inline static/app/views/monitors/details.tsx:50]: Going to be refactoring monitor head to use `Layout.Header` in a followup PR

File: static/app/views/monitors/create.tsx [hunk 2]
```diff
@@ -21,23 +21,25 @@ export default class CreateMonitor extends AsyncView<Props, AsyncView['state']>
...
   renderBody() {
     return (
-      <Fragment>
-        <h1>New Monitor</h1>
-        <HelpText>
-          {t(
-            `Creating a monitor will allow you to track the executions of a scheduled
+      <Layout.Body>
+        <Layout.Main fullWidth>
+          <h1>New Monitor</h1>
```
Comment (billyvg) [inline static/app/views/monitors/create.tsx:26]: `t()`

File: static/app/views/monitors/edit.tsx [hunk 3]
```diff
@@ -40,16 +40,18 @@ export default class EditMonitor extends AsyncView<Props, State> {
...
-        <MonitorForm
-          monitor={monitor}
-          apiMethod="PUT"
-          apiEndpoint={`/monitors/${monitor.id}/`}
-          onSubmitSuccess={this.onSubmitSuccess}
-        />
-      </Fragment>
+      <Layout.Body>
+        <Layout.Main fullWidth>
+          <h1>Edit Monitor</h1>
```
Comment (billyvg) [inline static/app/views/monitors/edit.tsx:45]: `t()`

**Review by billyvg — APPROVED** [review]
LGTM

**Pull request merged**